src/app/project/board/project-tab-view/project-tab-view.component.html

```diff
@@ -0,0 +1,94 @@
+<p>Metadata for selected project is listed below.</p>
+
+<div class="properties-container">
+    <div class="properties">
+        <div *ngFor="let prop of metadata | keyvalue; let last = last; let i = index;" class="border-bottom">
+            <div *ngIf="excludeKeys.indexOf(prop.key) < 0" class="property">
+                <div class="property-label">
+                    <h3 class="label mat-subheading-1">
+                        {{ prop.key }}
```

Comment:
is there a way to get the label instead of the key? because the keys are not well written (e.g. startDate, dataManagementPlan)
same question for the panel "Dataset"

Reply:
I agree but it requires changes in the backend. Currently I am using the [format](https://docs.google.com/document/d/129cFQIcFMWHjWLb1rvzN9kTcKO_MpGmJgjCzeiswNY4/edit) received from Js-lib. If we want to use label, we need to update this response format. 
I would suggest to update the response format (in backend) in next iteration. So that I can directly receive it in frontend using metadata endpoint and use it instead of hardcoding it now.

Reply:
ok,I think it is important to have it soon

Reply:
I'm not API designer specialist, but I think common practice is to use **lowerCamelCase** for properties to avoid spaces and other strange characters. I've never also worked with in this way designed API, however it's possible to find this this kind of ideas in the web, I'm not sure if this is best practice. @tobiasschweizer what do you think about it?

Reply:
Sorry, I think I am missing some context here. Could you give me some examples?

Reply:
I think I talked about it with you already. Snehal would like to have Space Separated properties instead of lowerCamelCase, as we have now, just to iterate through and display it directly on the view. But I've just understood it is rather not doable.

Reply:
You can surely have display strings (labels) that contain spaces, but property names cannot have spaces. So I think you are right about the camel case convention.

Reply:
Let's discuss this in next metadata discussion meeting. I am adding @BalduinLandolt for reference here as he is coordinating the metadata group.